voluncheer/.ebextensions/django.config

```diff
@@ -14,7 +14,7 @@ container_commands:
   05_remove_old_archive_cron:
     command: "rm -f /etc/cron.d/archive_opportunities"
   06_create_archive_cron:
-    command: "echo '0 0 * * * root source /var/app/venv/*/bin/activate && python3 manage.py archive_opportunities' > /etc/cron.d/archive_opportunities"
+    command: "echo '0 * * * * root source /var/app/venv/*/bin/activate && python3 manage.py archive_opportunities' > /etc/cron.d/archive_opportunities"
```

Comment:
How are you going to confirm this is running? Are you planning to create an opportunity you expect to be archived and then check it's status after an hour?

Reply:
Yes, that's what I was thinking. I actually just changed it to run every ten minutes so that I can check the results earlier

Reply:
SGTM